[2024-11-03T17:31:49.701+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-03T17:31:49.709+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.get_metadata manual__2024-11-03T17:31:27.123863+00:00 [queued]>
[2024-11-03T17:31:49.712+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.get_metadata manual__2024-11-03T17:31:27.123863+00:00 [queued]>
[2024-11-03T17:31:49.712+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 3
[2024-11-03T17:31:49.717+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): get_metadata> on 2024-11-03 17:31:27.123863+00:00
[2024-11-03T17:31:49.719+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=166) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-03T17:31:49.720+0000] {standard_task_runner.py:72} INFO - Started process 171 to run task
[2024-11-03T17:31:49.720+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'get_metadata', 'manual__2024-11-03T17:31:27.123863+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/***.py', '--cfg-path', '/tmp/tmpyh1_fk56']
[2024-11-03T17:31:49.721+0000] {standard_task_runner.py:105} INFO - Job 80: Subtask get_metadata
[2024-11-03T17:31:49.737+0000] {task_command.py:467} INFO - Running <TaskInstance: data_pipeline.get_metadata manual__2024-11-03T17:31:27.123863+00:00 [running]> on host 9ab0960d13ae
[2024-11-03T17:31:49.760+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='your_name' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='get_metadata' AIRFLOW_CTX_EXECUTION_DATE='2024-11-03T17:31:27.123863+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-03T17:31:27.123863+00:00'
[2024-11-03T17:31:49.760+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-03T17:31:51.491+0000] {python.py:240} INFO - Done. Returned value was: Done
[2024-11-03T17:31:51.512+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-03T17:31:51.513+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=get_metadata, run_id=manual__2024-11-03T17:31:27.123863+00:00, execution_date=20241103T173127, start_date=20241103T173149, end_date=20241103T173151
[2024-11-03T17:31:51.564+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-03T17:31:51.579+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-03T17:31:51.580+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
